src/derivative_operators/convolutions.jl

```diff
@@ -28,12 +37,27 @@ end
 function convolve_BC_left!(x_temp::AbstractVector{T}, x::AbstractVector{T}, A::DerivativeOperator) where {T<:Real}
     stencil = A.low_boundary_coefs
     coeff   = A.coefficients
-    for i in 1 : A.boundary_point_count
-        cur_stencil = stencil[i]
+
+    _bpc = A.boundary_point_count
+    flag = false
+    if stencil == []
```

Comment:
isempty

Reply:
if it's empty, wouldn't you just return?

Reply:
No, we have to still take care of the boundary point which and instead of boundary stencil, use the normal interior stencil.